Makefile

```diff
@@ -7,15 +7,19 @@ all: welcome
 	@exit 1
 
 doc:
+	export STRIP_FROM_INC_PATH_LIST=$(find . -type d -name include -printf " \"$PWD/%P\"")
 	"$(MAKE)" -BC doc/doxygen
```

Comment:
It's even worse. This doesn't work. :-/
Line 11  is another shell process as line 10.